addon/services/cookies.js

```diff
@@ -120,9 +120,15 @@ export default Ember.Service.extend({
   },
 
   _filterCachedFastbootCookies(fastbootCookiesCache) {
-    let request = this.get('_fastboot.request');
+    let request = this.get('_fastboot._fastbootInfo.request');
     return _collection.reduce(fastbootCookiesCache, (acc, cookie, name) => {
       let { value, options } = cookie;
+
+      if (!options) {
+        acc[name] = value;
+        return acc;
+      }
```

Comment:
I guess `options` should just default to `{}` in which case none of the below cases would apply and the cookie would be returned.